torchvision/prototype/transforms/_meta.py

```diff
@@ -66,3 +66,11 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         return F.convert_color_space(
             inpt, color_space=self.color_space, old_color_space=self.old_color_space, copy=self.copy
         )
+
+
+class ClampBoundingBox(Transform):
```

Comment:
Why isn't this private as well?

Reply:
Clamping boxes is useful in a general context, is it not? `torchvision.prototype.transforms.functional.clamp_bounding_box` is public, so shouldn't we also have a public transformation for it?

Reply:
Clamping is certainly useful but I would expect that one would do it by using directly the proposed clean up transform. It really depends on what idiom you would like to push for. I was imagining that we would be promoting the use of `CleanupBoxes` instead of individual components. WDYT?

Reply:
`_ClampBoundingBoxes`? Plural?